lib/symengine/basic.rb

```diff
@@ -11,8 +11,13 @@ def free_symbols
     end
     
     def abs
-        return SymEngine::abs(self)
+      SymEngine::abs(self)
+    end
+    def to_proc()
+      if self.free_symbols.count > 1
+        raise ArgumentError, "Too many free symbols! Only 1 allowed. Found #{self.free_symbols.count}."
```

Comment:
Aaaah, symbols ordering :( Got it. No, no magical way to add arguments we have, but explicit call may be good enough?..

```ruby
[[1, 10], [2,20]].map(&formula.to_proc(:h, :r))
```

So, it looks like `#to_proc` should accept list of symbols, and rais in case when there is more than one free symbol AND no symbol list passed. WDYT?